lint/problem.go

```diff
@@ -59,8 +59,6 @@ type Problem struct {
 
 	// The category for this problem, based on user configuration.
 	category string
-
-	noPositional struct{}
```

Comment:
This serve a purpose. We do not want to remove them.

Reply:
Reverted.